src/core/noelle/src/Noelle_loops.cpp

```diff
@@ -138,13 +137,14 @@ std::vector<LoopStructure *> *Noelle::getLoopStructures(double minimumHotness) {
      */
     auto loops = LI.getLoopsInPreorder();
     for (auto loop : loops) {
-      auto currentLoopIndex = nextLoopIndex++;
 
       /*
        * Check if the loop is hot enough.
        */
       auto loopStructure = new LoopStructure{ loop };
-      auto loopHeader = loopStructure->getHeader();
+      auto loopIDOpt = loopStructure->getID();
```

Comment:
Are we going to introduce the assumption that all loops must have an ID before using any NOELLE's API?
In other words, are we going to add loop IDs in noelle-norm?

Reply:
yes